src/raymath.h

```diff
@@ -1133,9 +1132,9 @@ RMAPI float MatrixDeterminant(Matrix mat)
     float result = 0.0f;
 
     // Cache the matrix values (speed optimization)
-    float a00 = mat.m0, a01 = mat.m1, a02 = mat.m2, a03 = mat.m3;
-    float a10 = mat.m4, a11 = mat.m5, a12 = mat.m6, a13 = mat.m7;
-    float a20 = mat.m8, a21 = mat.m9, a22 = mat.m10, a23 = mat.m11;
+    float a00 = mat.m0 , a01 = mat.m1 , a02 = mat.m2 , a03 = mat.m3;
```

Comment:
I prefer to avoid this kind of alignments based on additional spaces. One space is ok despite lines are not aligned.